recipes/authlib/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "authlib" %}
+{% set version = "0.10" %}
+{% set sha256 = "b61c6c6fd230c4ba8602fd85ee9a40e6dc859387699a1cd1f7247c4b109dcc17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/c8/03/6c676d80761f5194ef343679240bb3cfabb4122f9ee1c068ef83fec4ef7f/Authlib-0.10.tar.gz
```

Comment:
Please change this to:
```
url: https://files.pythonhosted.org/packages/c8/03/6c676d80761f5194ef343679240bb3cfabb4122f9ee1c068ef83fec4ef7f/Authlib-{{ version }}.tar.gz
```

Reply:
In that way when the version update our bot will be able to track this recipe and update it